package.json

```diff
@@ -3,7 +3,9 @@
   "private": true,
   "dependencies": {
     "datatables.net": "^1.10.15",
-    "datatables.net-dt": "^1.10.15",
+    "datatables.net-bs": "^1.10.20",
+    "datatables.net-dt": "^1.10.20",
```

Comment:
So, the question is, do you need the `-dt` package now?

Reply:
(You aren't including anything from it)

Reply:
Oh gosh right